Casks/vagrant.rb

```diff
@@ -0,0 +1,27 @@
+cask "hashicorp-vagrant" do
+  version "2.3.3"
+  sha256 ""
```

Comment:
do we need to hardcode the `sha256`?

Reply:
This should be auto-populated by the templater tool

Reply:
I couldn't get this fixed in ~30 mins, so it will take a little more debugging. In the `printFormula` function in `formula.go`, the shasums are supposed to be populated. That part doesn't seem to be working properly for vagrant